src/compileTree/compileTree.ts

```diff
@@ -71,5 +71,6 @@ export class CompileTree {
 }
 
 // Removes header from SAS file
+// Removes multiline comment in the begging of the string regardless of the white spaces or line breaks before it
```

Comment:
// If string begins with a multiline comment, remove the comment
// Ignore any preceding white space or line breaks